app/model/Layer.js

```diff
@@ -16,7 +16,7 @@ Ext.define('MoMo.admin.model.Layer', {
         name: 'name',
         type: 'string'
     }, {
-        name: 'type',
+        name: 'hoverFieldTemplate',
```

Comment:
I'm not 100% sure about this. I think that "type" is not exisiting in the backend so i removed it. Hovertemplate seems to be in the Appearance ... so i'll remov this.